src/sagas/deposit.js

```diff
@@ -95,6 +95,9 @@ function* restoreState(nextStepMap, stateKey) {
             // FIXME Check to see if we have a transaction in the mempool for
             // FIXME submitting funding proof, and update state accordingly.
 
+            // TODO Fork on active vs await
+            yield put(navigateTo('/deposit/' + depositAddress + nextStep))
```

Comment:
Hmmm, why did this land here when it's also below?

Reply:
Artifact from previous bugfix of what appeared to be a race. 

3b563bd1f656ba463b1f69b97af82b3800989e63 for reference, since we removed waitConfirmations, it no longer applies.

Will remove.